ARG queries/Container Registry Vulnerability Assessment Images Scan Summary/readme.md

```diff
@@ -0,0 +1,20 @@
+# Container Registry Images scan vulnerability assessment summary
+**Easy way to get Images summary for vulnerabilities scan, can be filtered for specific image or repository.**
```

Comment:
Maybe "How to get image vulnerability assessment summary that can be filtered by specific image or repository"